luatest/runner.lua

```diff
@@ -107,6 +108,8 @@ Options:
                           May be repeated to exclude several patterns
                           Make sure you escape magic chars like +? with %
   --coverage:             Use luacov to collect code coverage.
+  --no-clean:             Do not clean the working directory (/tmp/t).
```

Comment:
`Disable the var directory (default: /tmp/t) cleanup before running tests.`

Reply:
Fixed